planner/update.go

```diff
@@ -126,12 +131,7 @@ func (n *updateNode) simpleExplain() (map[string]any, error) {
 	}
 
 	// Add the attribute that represents the patch to update with.
-	data := map[string]any{}
-	err := json.Unmarshal([]byte(n.patch), &data)
-	if err != nil {
-		return nil, err
-	}
-	simpleExplainMap[dataLabel] = data
+	simpleExplainMap[dataLabel] = n.input
```

Comment:
todo: Similar to other suggestions please change the variable name to be more relevant.

```suggestion
	simpleExplainMap[inputLabel] = n.input
```